packages/components/src/alignment-matrix-control/utils.tsx

```diff
@@ -31,16 +35,28 @@ export const ALIGNMENT_LABEL: Record< AlignmentMatrixControlValue, string > = {
 export const ALIGNMENTS = GRID.flat();
 
 /**
- * Parses and transforms an incoming value to better match the alignment values
+ * Normalizes an incoming value to better match the alignment values
+ *
+ * @param value an alignment value to normalize
+ *
+ * @return The normalized value
+ */
+export function normalizeValue( value: VoidableAlignmentMatrixControlValue ) {
```

Comment:
We could avoid `export`-ing functions that are not used externally. 

```suggestion
function normalizeValue( value: VoidableAlignmentMatrixControlValue ) {
```